parser/opcode/opcode.go

```diff
@@ -134,5 +134,5 @@ var opsLiteral = map[Op]string{
 
 // Format the ExprNode into a Writer.
 func (o Op) Format(w io.Writer) {
-	fmt.Fprintf(w, opsLiteral[o])
+	fmt.Fprintf(w, "%s", opsLiteral[o])
```

Comment:
maybe `w.Write([]byte(opsLiteral[o]))` is better? 0.0?

Reply:
or make `opsLiteral` be `map[Op][]byte` then `w.write(opsLiteral[o])`

Reply:
There is still a memory allocation in `w.Write([]byte(opsLiteral[o]))` when converting string to []byte.
If you want better performance, the 'hack' could be used, but I don't think it matters here.